smoove/0.2.5/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+LABEL maintainer="Miguel Brown (brownm28@email.chop.edu)"
+LABEL description="Custom build of smoove from https://github.com/brentp/smoove/releases for control/consistency in sw versions"
+
+ENV LUMPY_VERSION 0.3.0
+ENV SAMTOOLS_VERSION 1.9
+ENV SVTYPER_VERSION 0.7.1
+ENV SMOOVE_VERSION 0.2.5
+ENV GSORT_VERSION 0.1.3
+ENV MOSDEPTH_VERSION 0.2.6
+ENV DUPHOLD_VERSION 0.2.1
+
+RUN apt update && apt install -y build-essential git cmake curl zip python python-pip tabix gawk autoconf libcurl4-openssl-dev libz-dev liblzma-dev libssl-dev bsdmainutils libncurses5-dev libbz2-dev bcftools
+RUN pip install pysam numpy==1.16.5
+RUN curl -OL https://github.com/samtools/samtools/releases/download/${SAMTOOLS_VERSION}/samtools-${SAMTOOLS_VERSION}.tar.bz2 && tar -xjf samtools-${SAMTOOLS_VERSION}.tar.bz2 && rm samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+&& cd /samtools-${SAMTOOLS_VERSION} && ./configure && make && make install
+RUN cd / && git clone https://github.com/GregoryFaust/samblaster && cd samblaster && make && cp samblaster /usr/local/bin/. && cd / && rm -rf samblaster
```

Comment:
looks like samblaster has releases. maybe we should use one of those to further lock the docker down.

Reply:
Hmm, good catch.  I'll do that, thanks!

Reply:
Added requested info in latest commit. Testing docker here since cython needed to be added:
https://cavatica.sbgenomics.com/u/zhangb1/kf-somatic-tools-test/tasks/fe6b19e4-6de1-42b7-889e-6a9e5d3dc8c1